src/Main.ts

```diff
@@ -540,6 +540,61 @@ export class Main {
         }
     }
 
+    private async fixDMMetadata(room: BridgedRoom) {
+        if (room.SlackType !== "im" || !room.SlackTeamId) {
```

Comment:
Not sure; I only put the falsy check since `room.SlackTeamId`'s type is `string | undefined` here.

I'm a fan of null-checked getters that are meant to be used after initialization & print a meaningful error if a property that should be set isn't.  But for now we can just trust the types.

Reply:
Aha, the room can be undefined if no message has been sent in it after a bridge restart.